### PR TITLE
fixed format error while running format script

### LINKED ISSUE #73: error while running format script
Getting the following error while running npm run format script 
```No files matching the pattern "'**/*.{js,jsx,ts,tsx}'" were found.```

### PR DESCRIPTION
**Pull request by kratos-respawned**
fixes #73 
fixed `No files matching the pattern "'**/*.{js,jsx,ts,tsx}'" were found.` error

### CODE REVIEW

**Comment by kratos-respawned** [comment]
fixes #73

File: cli/src/templates/base/package.json.ejs [hunk 1]
```diff
@@ -4,7 +4,7 @@
   "scripts": {
     "android": "expo start --android",
 	<% if (props.packageManager === "bun") { %>
-		"format": "bun run eslint '**/*.{js,jsx,ts,tsx}' --fix && bun run prettier '**/*.{js,jsx,ts,tsx,json}' --write",
+		"format": "bun run eslint **/*.{js,jsx,ts,tsx} --fix && bun run prettier **/*.{js,jsx,ts,tsx,json} --write",
```
Comment (danstepanov) [inline cli/src/templates/base/package.json.ejs:7]: We do formatting like this elsewhere so I wonder if this approach would resolve the issue. @catalinmiron any thoughts on this?

```suggestion
		"format": "bun run eslint \"**/*.{js,jsx,ts,tsx}\" --fix && prettier \"**/*.{js,jsx,ts,tsx,json}\" --write",
```

**Review by danstepanov — COMMENTED** [review]
Just want to clarify one detail before accepting and merging.

**Comment by danstepanov** [comment]
@kratos-respawned can you also run `bunx changeset`, select `patch`, provide a summary, and include it in your commit? We've implemented a new system for auto-generating releases which temporarily requires this additional step.

File: cli/src/templates/base/package.json.ejs (hunk 1 above)
Comment (kratos-respawned) [reply cli/src/templates/base/package.json.ejs]: done

Comment (catalinmiron) [reply cli/src/templates/base/package.json.ejs]: Not really, when copying the templates, we're running `npx` yarn dlx` `bunx` `pnpmx` via [getPackageManager.ts](https://github.com/danstepanov/create-expo-stack/blob/beta/cli/src/utilities/getPackageManager.ts). and run `eslint` globally. This change should affect the code after `noInstall=false`.  I think that this is a valid change.

PS: Haven't tested locally but @kratos-respawned could you please share the output of both `noInstall` true and false?

Thanks!

**Comment by kratos-respawned** [comment]
> @kratos-respawned can you also run `bunx changeset`, select `patch`, provide a summary, and include it in your commit? We've implemented a new system for auto-generating releases which temporarily requires this additional step.

I'm currently using a Windows system, and unfortunately, "bun" is not supported on this platform. Additionally, I'm still a bit of a newcomer to "bun," and I could use some guidance on how to navigate through this process. Could you please provide some assistance or alternative steps that would work on a Windows environment? Thanks

**Comment by sammoore** [comment]
> > @kratos-respawned can you also run `bunx changeset`, select `patch`, provide a summary, and include it in your commit? We've implemented a new system for auto-generating releases which temporarily requires this additional step.
> 
> I'm currently using a Windows system, and unfortunately, "bun" is not supported on this platform. Additionally, I'm still a bit of a newcomer to "bun," and I could use some guidance on how to navigate through this process. Could you please provide some assistance or alternative steps that would work on a Windows environment? Thanks

Windows development support is still underway, but for now you could include a file `.changeset/spicy-humans-fix.md` (FWIW, bun auto-generated this file name for me) with the following contents (replacing `PATCH_SUMMARY` with a summary of your patch):

```markdown
---
'create-expo-stack': patch
---

PATCH_SUMMARY
```

I believe this should be enough to satisfy the changeset requirements. cc @danstepanov

**Comment by sammoore** [comment]
Replaced by #83

**Pull request closed without merging**